src/services/users/get-invited-user-detail.test.js

```diff
@@ -0,0 +1,61 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const getInvitedUserDetail = require('./get-invited-user-detail')
+
+
+describe('GET invited-user information', () => {
+	
+	let invalidInviteToken
+
+	before(async () => {
+		
+		invalidInviteToken = fixtures.generateAccessToken({
```

Comment:
not sure we want this in a `before` block since i don't see it  being used in more than one test case

Reply:
great